src/sql/workbench/services/accountManagement/browser/accountPickerImpl.ts

```diff
@@ -193,6 +252,32 @@ export class AccountPicker extends Disposable {
 		return null;
 	}
 
+	private renderTenantLabel(container: HTMLElement): IDisposable | null {
+		if (container.hasChildNodes()) {
```

Comment:
This check should be unnecessary - if there's no child nodes the for loop will no-op

Reply:
There's also a helper function to do this : https://github.com/microsoft/azuredatastudio/blob/main/src/vs/base/browser/dom.ts#L20

Reply:
It's a direct copy paste from the accountLabel one - should I still change it?

Reply:
Might as well if we're doing this kind of refactor anyways IMO